server/src/main/java/io/deephaven/server/table/ops/TimeTableGrpcImpl.java

```diff
@@ -48,9 +51,27 @@ public void validateRequest(final TimeTableRequest request) throws StatusRuntime
     public Table create(final TimeTableRequest request,
             final List<SessionState.ExportObject<Table>> sourceTables) {
         Assert.eq(sourceTables.size(), "sourceTables.size()", 0);
+        if (request.hasStartTimeString()) {
```

Comment:
Not a fan of these divergent construction sites. I'm assuming we should be able to adapt the types as necessary and have a single construction call.